T3_4/src/RomanNumeralTest.java

```diff
@@ -0,0 +1,29 @@
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class RomanNumeralTest {
+
+	@Test
+	public void exampleRomanToNumeral1() {
+		RomanNumeral rn = new RomanNumeral("X");
+		assertEquals(10, rn.toInt());
+	}
+	
+	@Test
+	public void exampleRomanToNumeral2() {
+		RomanNumeral rn = new RomanNumeral("XXIII");
+		assertEquals(23, rn.toInt());
+	}
+
+	@Test
+	public void exampleNumeralToRoman1(){
+		RomanNumeral rn = new RomanNumeral(10);
+		assertEquals("X", rn.toString());
+	}
+	@Test
+	public void exampleNumeralToRoman2(){
+		RomanNumeral rn = new RomanNumeral(324);
+		assertEquals("CCCXXIV", rn.toString());
```

Comment:
Consider add cases more significatives like: empty, zero, non-legal characters, etc. in order to test for robustness.
TDD philosophy is also think how can I do to fail it.